### DISCUSSION TITLE
[Newbie needs help] Instagram authorization headers

### DISCUSSION THREAD

**Discussion by potatomilk23** [AAndyProgram/SCrawler#57] [Q&A] [answered]
Hi, I have trouble getting the Instagram authorization headers. I followed every steps but I can't seem to find the required values. I'm not sure what I'm doing wrong. I'd really appreciate it if someone could provide some screenshots/video as I'm really confused right now :(

Thanks!

**Comment by AAndyProgram** [comment, marked as answer]
Sorry, but I don't know how to help you. How to find [Hash2](https://github.com/AAndyProgram/SCrawler/wiki/Settings#how-to-find-instagram-hash-2) and [tokens](https://github.com/AAndyProgram/SCrawler/wiki/Settings#how-to-find-instagram-authorization-headers) is described in the wiki. Be sure to follow **step 3 & 5** of these instructions: **if this tab contain any data, clear it**.

**Reply by MixingMA** [reply]
I had the same problem. After following steps 1-4, searching for "x-ig-app-id: and "x-ig-www-claim" under Network in the tagged photos tab doesn't turn up anything

**Reply by AAndyProgram** [reply]
Try reloading the page again. Sometimes you need to reload the page twice.

**Reply by MixingMA** [reply]
The issue remains after reloading multiple times

![image](https://user-images.githubusercontent.com/109399051/179379668-6f3bf46d-8f9d-49bc-8fb4-be48e3355e0f.png)

**Reply by AAndyProgram** [reply]
1. It looks like Mozilla Dev tools. Try using Google Chrome.
2. It's impossible not to have records of network activity on page reload. At least one file must be there. If you don't see anything, you are doing something wrong.

**Reply by MixingMA** [reply]
Chrome is showing the same thing. There's entries in network activity, just nothing that pops up when searching "x-ig", the screenshot is showing an attempt to search for "x-ig" under network activity.

**Reply by AAndyProgram** [reply]
**Excuse me, but I just now saw this: you are using a filter, not a search.**
----
![image](https://user-images.githubusercontent.com/88590076/179527685-317e943c-2b0e-4add-aed4-4609f61e6922.png)
